dds/DCPS/LinuxNetworkConfigMonitor.cpp

```diff
@@ -50,7 +50,7 @@ bool LinuxNetworkConfigMonitor::open()
   struct {
     nlmsghdr header;
     rtgenmsg msg;
-  } request = {};
+  } request = { {}, {} };
```

Comment:
See #2030

Reply:
[This conversion in particular](https://github.com/objectcomputing/OpenDDS/pull/2030/files#r520194627)

Reply:
ok, I'll back this out.

Reply:
Well now that I think about it, that's for XTypes, but it should probably be brought into master for 3.15...